.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Ruby CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    environment: testing
+    env:
+      DATABASE_NAME_TESTING: ${{ secrets.DATABASE_NAME_TESTING }}
+      DATABASE_USER_TESTING: ${{ secrets.DATABASE_USER_TESTING }}
+      DATABASE_PASSWORD_TESTING: ${{ secrets.DATABASE_PASSWORD_TESTING }}
+      DATABASE_URL_TESTING: "postgres://${{secrets.DATABASE_USER_TESTING }}:${{secrets.DATABASE_PASSWORD_TESTING}}@localhost/postgres_testing"
+
+      DATABASE_NAME: ${{ secrets.DATABASE_NAME }}
+      DATABASE_USER: ${{ secrets.DATABASE_USER }}
+      DATABASE_PASSWORD: ${{ secrets.DATABASE_PASSWORD }}
+      DATABASE_URL: "postgres://${{secrets.DATABASE_USER }}:${{secrets.DATABASE_PASSWORD}}@localhost/training_dev"
```

Comment:
Is this one needed?

Reply:
They are not used but are needed to pass the action. I think it's a problem of the database configuration. I tried to fix it but was unsuccessful.